spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/controller/PackageController.java

```diff
@@ -108,13 +106,22 @@ public void packageDelete(@PathVariable("name") String name) {
 		this.packageMetadataService.deleteIfAllReleasesDeleted(name, PackageMetadataService.DEFAULT_RELEASE_ACTIVITY_CHECK);
 	}
 
+	@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "Release not found")
```

Comment:
Maybe we can do some refactoring by having the same exception handlers in both the controllers. I guess this is fine for now.

Reply:
That would be the global error handling. Don't wanna go there for now as it's a bit more dark territory.